examples/algebraic_constraints.py

```diff
@@ -0,0 +1,44 @@
+import desbordante
+import pandas
+
+TABLE = 'datasets/cargo_march.csv'
+HEADER = 0
+SEPARATOR = ','
+P_FUZZ = 0.85
+FUZZINESS = 0.2
+BUMPS_LIMIT = 0
+WEIGHT = 0.1
+BIN_OPERATION = '-'
+AC_SEED = 11
+ITERATIONS_LIMIT = 4
+
+algo = desbordante.ACAlgorithm()
+
+df = pandas.read_csv(TABLE, sep=SEPARATOR, header=HEADER)
+df_without_id = df[['Delivery date', 'Dispatch date']]
+
+print("Algorithm needs the following data format:", end=' ')
+print(algo.get_needed_options())
+algo.load_data(df=df_without_id)
+
+print("Algorithm needs the following options:", end=' ')
+print(algo.get_needed_options())
```

Comment:
```suggestion
algo.load_data(df_without_id)
```
I don't think those are needed. Printing the options you are using, on the other hand, makes more sense.

Reply:
We left it because we wanted somewhere an example on how to get options for the algorithm